java/classfile/arch.xml

```diff
@@ -54,7 +54,7 @@
          category="stable">
     </api>
    The ClassFile library is an API to read Java classfiles, as described by
-   <a href="http://java.sun.com/docs/books/vmspec/2nd-edition/html/VMSpecTOC.doc.html">
+   <a href="https://docs.oracle.com/javase/specs/jvms/se6/html/VMSpecTOC.doc.html">
```

Comment:
Better use a newer version, at least version 8? `https://docs.oracle.com/javase/specs/jvms/se8/html/index.html`

Reply:
why not the background blue doc looks a bit old.